### PR TITLE
Use debug option in snapctl package

### PR DESCRIPTION
**Pull request by farshidtz**
This is to allow turning on debug mode for all modules using `snap set <snap> debug=true`.

### CODE REVIEW

File: log/init.go [hunk 1]
```diff
@@ -17,12 +17,12 @@ func init() {
 }
 
 func initialize() {
-	value, err := snapctl.Get("debug").Run()
+	value, err := exec.Command("snapctl", "get", "debug").CombinedOutput()
```
Comment (farshidtz) [inline log/init.go:20]: Executing command instead of using snapctl package to avoid circular dependency imports.

Comment (MonicaisHer) [reply log/init.go:20]: Are package `snapctl` and `log` imported from each other?

Comment (farshidtz) [reply log/init.go:20]: I have made this particular change so that they don't import each other.

**Comment by MonicaisHer** [comment]
```
go: downloading github.com/farshidtz/edgex-snap-hooks/v2 v2.0.6-0.20220420071117-524c46f8ae82
...
$ snap set edgex-device-mqtt debug=true
```
```
$ journalctl -f | grep device-mqtt
...
Apr 20 09:45:22 mengyi-computer edgex-device-mqtt[281176]: Changing syslog tag to: edgex-device-mqtt.configure
Apr 20 09:45:22 mengyi-computer edgex-device-mqtt.configure[281176]: Executing 'snapctl get env.security-bootstrapper.add-registry-acl-roles'
Apr 20 09:45:22 mengyi-computer edgex-device-mqtt.configure[281176]: Executing 'snapctl get env.security-secret-store.add-secretstore-tokens'
Apr 20 09:45:22 mengyi-computer edgex-device-mqtt.configure[281176]: Executing 'snapctl get env.security-secret-store.add-known-secrets'
Apr 20 09:45:22 mengyi-computer edgex-device-mqtt.configure[281176]: Executing 'snapctl unset env.security-bootstrapper'
Apr 20 09:45:22 mengyi-computer edgex-device-mqtt.configure[281176]: Executing 'snapctl unset env.security-secret-store'
Apr 20 09:45:22 mengyi-computer edgex-device-mqtt.configure[281176]: Executing 'snapctl get env'
Apr 20 09:45:22 mengyi-computer edgex-device-mqtt.configure[281176]: Executing 'snapctl get -d config'
Apr 20 09:45:22 mengyi-computer edgex-device-mqtt.configure[281176]: No global configuration settings
Apr 20 09:45:22 mengyi-computer edgex-device-mqtt.configure[281176]: Executing 'snapctl get -d apps'
Apr 20 09:45:22 mengyi-computer edgex-device-mqtt.configure[281176]: Processing service:device-mqtt
Apr 20 09:45:22 mengyi-computer edgex-device-mqtt.configure[281176]: Processing appConfig:map[]
...
```

**Review by MonicaisHer — APPROVED** [review]

**Comment by farshidtz** [comment]
> Is that necessary also print the outputs of snapctl for debugging?

We can add in future if needed. This PR was about making debug mode consistent accross modules.

**Pull request merged**